test/index/down.test.js

```diff
@@ -360,4 +347,40 @@ describe('down', function () {
       });
     });
   });
+};
+
+describe('down', function () {
+  beforeEach(function () {
+    return helper
+      .prepareMigrations(3)
+      .bind(this)
+      .then(function (migrationNames) {
+        this.migrationNames = migrationNames;
+        this.umzug          = new Umzug({
+          migrations:     { path: __dirname + '/../tmp/' },
+          storageOptions: { path: __dirname + '/../tmp/umzug.json' }
+        });
+      });
+  });
+
+  downTestSuite();
+
+});
+
+describe('down-directories', function () {
+  beforeEach(function () {
+    return helper
+      .prepareMigrations(3, {directories: [['1', '2'], ['1', '2'], ['1', '3', '4', '5']]})
```

Comment:
No. It checks if it exists and then adds it. This is done via the helper. These are directories and not file names. Every file you want to add (in this case, 3) should have a directory to add it too, otherwise, it will be in the root tmp directory. So, in this example there will be two files in 'tmp/1/2/' directory and one in 'tmp/1/3/4/5/' directory. The file and file names work as before, the change was to put them in directories.